README.md

```diff
@@ -8,34 +8,318 @@ or
 `yarn add teepublic-react`
 
 # Quick Start
-These components are designed for responses from the TeePublic.com API.  With an API response from  `/v1/stores/{id}`, you can use the DesignCollection component to quickly show tiles.
+# For Ruby and React Stack
+## Step 1 - Get TeePublic Gem
+Add TeePublic gem to your Gemfile, which is a wrapper around TeePublic APIs (https://api.teepublic.com/v1/docs#/)
+
+`gem 'teepublic-api', github: 'bustedtees/teepublic-api'`
+
+and run
+
+$`bundle install`
+
+## Step 2 - Configure
+Add `config/initializers/tee_public.rb`
+and copy paste the content below with your api key
+```
+require 'tee_public/api'
+
+TeePublic::Api.configure do |config|
+  config.api_key = 'YOUR_API_KEY_GOES_HERE'
+end
+```
+
+## Step 3 - Explore APIs
+Open rails console `rails c` and try these
+#### List of Stores API
+```
+  fetch_stores = TeePublic::Api.send('stores')
+  stores = fetch_stores.dig('body', '_embedded', 'stores').map(&:deep_symbolize_keys!)
+  puts stores
+```
+
+#### Single Store API
+```
+  # store_id -> Pick a store_id returned by List of Stores API
+  # page -> For trial purposes, you can use page = 1
+  store_url = "stores/#{store_id}?page=#{page}"
+
+  # album_id -> it's an additional filter option, can keep it blank for now,
+  # we will learn more about this later on
+  store_url += "&album_id=#{album_id}" if album_id.present?
+
+  # product_type -> same explanation as album_id
+  store_url += "&product_type=#{product_type}" if product_type.present?
+
+  fetch_store = TeePublic::Api.send(store_url).deep_symbolize_keys!    
+  store = fetch_store[:body]
+  puts store
+```
+
+#### Design API
+If no designs are found in Single Store API that means you have not added any designs to that store on teepublic.com. Head to TeePublic.com, locate that store from slug, '/stores/<store-slug>'. Add few designs to it.
+```
+  # design_id -> Single Store API returns a list of designs. Pick any design_id you like
+  fetch_design = TeePublic::Api.designs(design_id).deep_symbolize_keys!
+  design = fetch_design[:body]
+  puts design
+```
+
+#### SKUs API
+Each design can be printed on different products like t-shirt, mug or case. Further more, each product can have different skus like t-shirt colors, sizes, etc. Skus are different variants of a product for a given design
```

Comment:
Wording here is a little tricky.  Try this:
"TeePublic prints each design on a variety of products like t-shirts, mugs and phone cases.  Each product is of a certain product type, and each specific product (a combination of size, color, product type, etc) is represented by a single SKU"

Reply:
NICE! Copying it, this is what happens when someone with non-english background write docs